src/ReinforcementLearningCore/src/utils/distributions.jl

```diff
@@ -63,13 +61,15 @@ action_samples x batch_size).  Return a 3D matrix of size (1 x action_samples x
 batch_size). 
 """
 function mvnormlogpdf(μ::A, LorU::A, x::A; ϵ=1.0f-8) where {A<:AbstractArray}
-    logp = [mvnormlogpdf(μ[:, :, k], LorU[:, :, k], x[:, :, k]) for k in 1:size(x, 3)]
-    return unsqueeze(stack(logp; dims=2), dims=1) #returns a 3D vector 
+    l(k) = mvnormlogpdf(μ[:, :, k], LorU[:, :, k], x[:, :, k])
+    logp = reduce(hcat, [l(k) for k in axes(x, 3)])
```

Comment:
Want to use a tuple here instead of a vect?

```suggestion
    logp = reduce(hcat, (l(k) for k in axes(x, 3)))
```